<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QOperationLogWgt</class>
 <widget class="QWidget" name="QOperationLogWgt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2003</width>
    <height>835</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QOperationLogWgt</string>
  </property>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1912</width>
     <height>793</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1505</width>
     <height>0</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="spacing">
     <number>4</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="widget1" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="widget_4" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>59</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>1912</width>
           <height>59</height>
          </size>
         </property>
         <property name="bordersB" stdset="0">
          <string>bottom</string>
         </property>
         <property name="bks" stdset="0">
          <string>gray</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>20</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <property name="spacing">
             <number>20</number>
            </property>
            <item>
             <widget class="QWidget" name="widget" native="true">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>59</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>59</height>
               </size>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <property name="spacing">
                <number>40</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_4">
                 <property name="spacing">
                  <number>15</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="label_6">
                   <property name="text">
                    <string>用户名</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="user_edit">
                   <property name="minimumSize">
                    <size>
                     <width>150</width>
                     <height>40</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>150</width>
                     <height>40</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                 <property name="spacing">
                  <number>15</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="label_4">
                   <property name="text">
                    <string>操作类型</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QComboBox" name="operationType_cobox">
                   <property name="minimumSize">
                    <size>
                     <width>150</width>
                     <height>40</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>150</width>
                     <height>40</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_12">
                 <property name="spacing">
                  <number>15</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="label_12">
                   <property name="text">
                    <string>操作时间</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_13">
                   <property name="spacing">
                    <number>10</number>
                   </property>
                   <item>
                    <widget class="QDateEdit" name="lower_date_edit">
                     <property name="minimumSize">
                      <size>
                       <width>180</width>
                       <height>0</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>180</width>
                       <height>40</height>
                      </size>
                     </property>
                     <property name="wrapping">
                      <bool>false</bool>
                     </property>
                     <property name="buttonSymbols">
                      <enum>QAbstractSpinBox::NoButtons</enum>
                     </property>
                     <property name="showGroupSeparator" stdset="0">
                      <bool>false</bool>
                     </property>
                     <property name="currentSection">
                      <enum>QDateTimeEdit::YearSection</enum>
                     </property>
                     <property name="displayFormat">
                      <string>yyyy/MM/dd</string>
                     </property>
                     <property name="calendarPopup">
                      <bool>true</bool>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="label_13">
                     <property name="text">
                      <string>-</string>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignCenter</set>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QDateEdit" name="upper_date_edit">
                     <property name="minimumSize">
                      <size>
                       <width>170</width>
                       <height>40</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>180</width>
                       <height>40</height>
                      </size>
                     </property>
                     <property name="styleSheet">
                      <string notr="true"/>
                     </property>
                     <property name="buttonSymbols">
                      <enum>QAbstractSpinBox::NoButtons</enum>
                     </property>
                     <property name="currentSection">
                      <enum>QDateTimeEdit::YearSection</enum>
                     </property>
                     <property name="displayFormat">
                      <string>yyyy/MM/dd</string>
                     </property>
                     <property name="calendarPopup">
                      <bool>true</bool>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <property name="spacing">
               <number>10</number>
              </property>
              <item>
               <widget class="QPushButton" name="query_btn">
                <property name="minimumSize">
                 <size>
                  <width>100</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>100</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="text">
                 <string>查询</string>
                </property>
                <property name="buttons" stdset="0">
                 <string notr="true">flat_btn</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="reset_btn">
                <property name="minimumSize">
                 <size>
                  <width>100</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>100</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="text">
                 <string>重置</string>
                </property>
                <property name="buttons" stdset="0">
                 <string notr="true">flat_btn</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="expand_btn">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>58</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>58</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QTableView" name="tableView">
         <property name="maximumSize">
          <size>
           <width>1912</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">#tableView QScrollBar::handle:vertical  {
    margin-top: 46px;
}

#tableView QScrollBar::sub-line:vertical  {
    height: 43px; 
    border-bottom: 1px solid #a4a4a4;
}
</string>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAsNeeded</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAsNeeded</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="verticalScrollMode">
          <enum>QAbstractItemView::ScrollPerPixel</enum>
         </property>
         <property name="horizontalScrollMode">
          <enum>QAbstractItemView::ScrollPerPixel</enum>
         </property>
         <property name="showGrid">
          <bool>false</bool>
         </property>
         <property name="scrolls" stdset="0">
          <string notr="true">layout</string>
         </property>
         <property name="bordersR" stdset="0">
          <string notr="true">right</string>
         </property>
         <attribute name="horizontalHeaderHighlightSections">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderDefaultSectionSize">
          <number>44</number>
         </attribute>
         <attribute name="verticalHeaderMinimumSectionSize">
          <number>44</number>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_3" native="true">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="bks" stdset="0">
       <string notr="true">gray</string>
      </property>
      <property name="bordersL" stdset="0">
       <string notr="true">left</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>15</number>
       </property>
       <property name="leftMargin">
        <number>20</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>20</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_10">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>操作记录：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="record_label">
         <property name="minimumSize">
          <size>
           <width>340</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>360</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
